README.md

```diff
@@ -5,8 +5,8 @@ A browser interface based on Gradio library for Stable Diffusion.
 
 ## Features
 [Detailed feature showcase with images](https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Features):
-- Original txt2img and img2img modes
-- One click install and run script (but you still must install python and git)
+- Original txt to img and img to txt modes

```

Comment:
the term was used by the original implementation https://github.com/CompVis/stable-diffusion